coremltools/converters/mil/frontend/torch/test/test_executorch_quantization.py

```diff
@@ -0,0 +1,56 @@
+#  Copyright (c) 2023, Apple Inc. All rights reserved.
+#
+#  Use of this source code is governed by a BSD-3-clause license that can be
+#  found in the LICENSE.txt file or at https://opensource.org/licenses/BSD-3-Clause
+
+import pytest
```

Comment:
I think instead of creating a new file,
given the scope of testing converge of edgeir is not big yet,
i would suggest to just put this test under:
https://github.com/apple/coremltools/blob/main/coremltools/converters/mil/frontend/torch/test/test_executorch_e2e.py
In the future we can do further refactor, if more and more quantization models come out

Reply:
I would prefer to keep quantization in a separate file:
1. We separate quantization from normal, e.g. we have `ops` and `quantization_ops`
2. EXIR quantized model is few for now, but will grow soon